pkg/apiserver/registry/stats/nodelatencystats/rest_test.go

```diff
@@ -13,3 +13,200 @@
 // limitations under the License.
 
 package nodelatencystats
+
+import (
+	"context"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"k8s.io/apimachinery/pkg/api/errors"
+	"k8s.io/apimachinery/pkg/apis/meta/internalversion"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+
+	statsv1alpha1 "antrea.io/antrea/pkg/apis/stats/v1alpha1"
+)
+
+func TestREST(t *testing.T) {
+	r := NewREST()
+	assert.Equal(t, &statsv1alpha1.NodeLatencyStats{}, r.New())
+	assert.Equal(t, &statsv1alpha1.NodeLatencyStats{}, r.NewList())
+	assert.False(t, r.NamespaceScoped())
+}
+
+func TestRESTCreate(t *testing.T) {
+	summary := &statsv1alpha1.NodeLatencyStats{
+		ObjectMeta:           metav1.ObjectMeta{Name: "node1"},
+		PeerNodeLatencyStats: nil,
+	}
+	expectedObj := &statsv1alpha1.NodeLatencyStats{
+		ObjectMeta:           metav1.ObjectMeta{Name: "node1"},
+		PeerNodeLatencyStats: nil,
+	}
+
+	r := NewREST()
+	ctx, cancel := context.WithCancel(context.Background())
+	defer cancel()
```

Comment:
There is no need to create a cancellable context, you should just use `ctx := context.Background()`

same comment for other tests

Reply:
Ok, I've updated all of them.